### ISSUE TITLE
Incorrect "invalid indentation style" error

### ISSUE THREAD

**Issue by PaulSonOfLars** [yoheimuta/protolint#99]
## Issue:
`protolint` reports an invalid indentation error, which breaks protos when using the `--fix` flag.

## Expected outcome:
Valid proto output, as well as a correct error. This is not an indentation error, but a formatting error. The linter should be recommending the removal of the `;` after the enum close.

## Versions:
OS: `macOS Mojave 10.14.6`
protolint: `0.20.0` and `0.21.0`

## Steps to reproduce:

`protolint lint test.proto` where `test.proto` is:

```proto
syntax = "proto3";

package test;

message Test {
  enum State {
    UNKNOWN_STATE = 0;
    CLEAR = 1;
    PULLED = 2;
    STARTED = 3;
  };
}

message TestResponse {
}

service RunnerService {
  rpc ReceiveTest (Test) returns (TestResponse);
}
```

Protolint reports `[test.proto:11:4] Found an incorrect indentation style "  }". "  " is correct.`, which breaks proto syntax.

**Comment by yoheimuta** [comment]
@PaulSonOfLars Thank you for reporting the issue!
I fixed it and shipped v0.21.1.
Please confirm it works well whenever you have time.

**Comment by PaulSonOfLars** [comment]
Great, thank you for the speedy fix! Just tested, can confirm the issue is gone.
Thanks again!
